runtime/buildtools.mk

```diff
@@ -258,4 +261,8 @@ endif
 makeLibDir :
 	mkdir -p lib
 
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+.PHONY : j9includegen
```

Comment:
The condition is inverted: please just move the `.PHONY :` line before the declaration of `j9includegen`.